### ISSUE TITLE
Code Feature: Implementation of CI/CD

### ISSUE THREAD

**Issue by prathamshroff** [QasimWani/LeetHub#530]
Reviewing and manually checking every pull request would always be time-consuming and a pain if the code doesn't even end up working or compiling so why not implement simple compile & build checks right into the repo? Compute Time is also free with GitHub Actions and any large-scale open-source project utilizes it.

I would be happy to work on this DevOps Feature if you can enable permissions for me. Have a lot more ideas and would love to chat more as well if you're still looking for maintainers.

**Comment by QasimWani** [comment]
yes please. there's already 200+ issues and I have been overloaded with work. lmk of best practices.
